### DISCUSSION TITLE
Auth and Header config set on the Collection level

### DISCUSSION THREAD

**Discussion by xaun** [Kong/insomnia#4478] [Ideas]
It would be great to be able to configure auth and header options in collection settings that gets applied to all requests in that collection. This is useful for things like API keys and API versions.

**Comment by jackkav** [comment]
Add the key `DEFAULT_HEADERS` to your environment and foo will be sent as a header in all requests associated with this collection.
<img width="226" alt="image" src="https://github.com/Kong/insomnia/assets/3679927/5774503a-f70c-44a6-9bdb-81e34984a2ef">

**Comment by subnetmarco** [comment]
This is possible to also setup as a folder-level configuration.
